README.md

```diff
@@ -16,7 +16,7 @@ It powers executable Go scripts and plugins, in embedded interpreters or interac
 * Simple interpreter API: `New()`, `Eval()`, `Use()`
 * Works everywhere Go works
 * All Go & runtime resources accessible from script (with control)
-* Security: `unsafe` and `syscall` packages not used or exported by default
+* Security: `unsafe` and `syscall` packages are neither used nor exported by default
```

Comment:
This change is not related to the purpose of the PR, and will generate conflicts on my side. Please cancel it.